jax/_src/nn/initializers.py

```diff
@@ -129,15 +130,15 @@ def _compute_fans(shape: core.NamedShape, in_axis=-2, out_axis=-1,
   if isinstance(in_axis, int):
     in_size = shape[in_axis]
   else:
-    in_size = int(np.prod([shape[i] for i in in_axis]))
+    in_size = operator.index(prod(shape[i] for i in in_axis))
```

Comment:
For each of these changes: it's not clear to me why it's warranted / desired. Can you say more about what use-case motivated the change?

Reply:
Ah, it doesn't change any functionality, thus there isn't a use-case...
I think I can remove the `operator.index`, just use `prod`.

Reply:
Now it is a chore simplification PR. WDYT?

Reply:
I think the principle at play here is "if it's not broken, don't fix it". Every change to the codebase has a small but non-zero risk of creating an unintended consequence, so we should avoid making changes unless there is a good reason to do so.

Reply:
> Though in theory the `int(list.index())` to `list.index()` change is zero risk

Agreed, but if we're certain the new code does nothing, why add it?

I think this PR could probably be closed.